frontend/src/lib/components/statistics/LearningKitStatisticCard.svelte

```diff
@@ -1,44 +1,79 @@
 <script lang="ts">
 	import { Progress } from '@skeletonlabs/skeleton-svelte';
 	import { _ } from 'svelte-i18n';
-	import type { KitWithProgress } from '$lib/types/statistics';
+	import type { LearningKitRes } from '$lib/schemas/response/LearningKitRes';
+	import { createQuery } from '@tanstack/svelte-query';
+	import { api } from '$lib/api/apiClient';
+	import { getLearningKitById } from '$lib/api/collections/learningKit';
+	import { SquareArrowOutUpRight } from 'lucide-svelte';
 
-	interface Props {
-		kit: KitWithProgress;
+	interface LearningKitStatisticCardProps {
+		learningKitId: string;
 	}
 
-	const { kit }: Props = $props();
+	const { learningKitId }: LearningKitStatisticCardProps = $props();
+
+	const kitQuery = createQuery<LearningKitRes>({
+		queryKey: ['learning-kit-stats-detail', learningKitId],
```

Comment:
getLearningKitById but you use learning-kit-stats-detail key. I think it should be the same as when you use getLearningKitById (learning-kit?) so data invalidation works here too